pkg/server/auth/jwt.go

```diff
@@ -74,8 +74,6 @@ func NewJWTAuthorizationHeaderPrincipalGetter(log logr.Logger, verifier tokenVer
 }
 
 func (pg *JWTAuthorizationHeaderPrincipalGetter) Principal(r *http.Request) (*UserPrincipal, error) {
-	pg.log.Info("attempt to read token from auth header")
```

Comment:
This could be a debug level one, and we should ensure that all the `Principal` implementations have very similar calls so we can see what we're trying to do to authenticate.